packages/v4/patternfly-docs/content/design-guidelines/components/toolbar/toolbar.md

```diff
@@ -37,7 +37,22 @@ A search filter is intended to be used with a filter group as shown below when y
 ### Button group
 <img src="./img/button-group.png" alt="button group" width="205" />
 
- Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. No more than two items should be exposed as buttons. If you have more than two items, use an [overflow menu](#overflow-menu) component to save space.
+Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. Be selective about which actions to surface on the toolbar, exposing no more than two buttons and reserving the rest for an [overflow menu](#overflow-menu) component to save space.
+
+ **One or two clickable actions**
+ <img src= "./img/primary-button.png" alt= "single button" width="196"/>
+* Use a primary button for any call to action you want a user to take
+* In the case that using the action represented should not be encouraged, opt for a secondary button
+
+ **Three clickable actions**
+<img src= "./img/three-as-a-group.png" alt= "three actions" width="196"/>
+ * In the specific case where three actions are needed, the suggested guidance is to opt for an overflow menu with two items. This will prevent the toolbar from becoming too crowded, and also prevent the overflow menu from containing just one item. 
+
+ **More than three clickable actions**
+ <img src= "./img/greater-than-three.png" alt= "primary and secondary button" width="196"/>
+1. Use a primary button to indicate the most important action within the table
```

Comment:
Also this is a numbered list, but the other ones are just bullet points. Make this a bulleted list to match!